styles.css

```diff
@@ -66,25 +66,351 @@
+    width: 350px;
+    height: 52px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 26px;
+    text-align: center;
+}
+/* second-row */
+.features-list-secondrow {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 30px;
+    width: 1110px;
+    height: 148px;    
+}
+.secondrow-firstcontent {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    width: 350px;
+    height: 148px;
+    border-radius: 8px;
+}
+.secondrow-firstcontent-icon {
+    width: 32px;
+    height: 32px;
+}
+.secondrow-firstcontent-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0px;
+    gap: 8px;
+    width: 350px;
+    height: 92px;
+}
+.secondrow-firstcontent-main {
+    width: 350px;
+    height: 32px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+}
+.secondrow-firstcontent-sub {
+    width: 350px;
+    height: 52px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 26px;
+    text-align: center;
+}
+.secondrow-secondcontent {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    width: 350px;
+    height: 148px;
+    border-radius: 8px;
+}
+.secondrow-secondcontent-icon {
+    width: 32px;
+    height: 32px;
+}
+.secondrow-secondcontent-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0px;
+    gap: 8px;
+    width: 350px;
+    height: 92px;
+}
+.secondrow-secondcontent-main {
+    width: 350px;
+    height: 32px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+}
+.secondrow-secondcontent-sub {
+    width: 350px;
+    height: 52px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 26px;
+    text-align: center;
+}
+.secondrow-thirdcontent {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    width: 350px;
+    height: 148px;
+    border-radius: 8px;
+}
+.secondrow-thirdcontent-icon {
+    width: 32px;
+    height: 32px;
+}
+.secondrow-thirdcontent-text {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0px;
+    gap: 8px;
+    width: 350px;
+    height: 92px;
+}
+.secondrow-thirdcontent-main {
+    width: 350px;
+    height: 32px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+}
+.secondrow-thirdcontent-sub {
+    width: 350px;
+    height: 52px;
+    font-family: 'Inter', sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 26px;
+    text-align: center;
+}
```

Comment:
The code here is repetitive. In a way, it's manual. meaning if a third row of features were to be added, you would
go in and individually apply the same styles. the solution for this is to use one class name for:
- the features row
- another for the features card
- and another for the various elements within the card.

this way, if you create a new row of features, you only need to add the class name and it will auto assume the
styles. same for the cards and it's contents

- [ ] Optimise the CSS for feature cards